packages/maskbook/src/extension/popups/components/StyledInput/index.tsx

```diff
@@ -3,18 +3,20 @@ import { TextFieldProps, TextField } from '@material-ui/core'
 import { makeStyles } from '@masknet/theme'
 import { useStylesExtends } from '@masknet/shared'
 
-const useStyles = makeStyles()({
+const useStyles = makeStyles()((theme) => ({
     textField: {
         width: '100%',
     },
     textFieldInput: {
-        backgroundColor: '#F7F9FA',
+        backgroundColor: theme.palette.mode === 'light' ? '#F7F9FA' : theme.palette.background.default,
```

Comment:
Use color constant to replace `#F7F9FA`?

Reply:
`theme.palette.background.default` is `#f9fafa`, very close to  `#F7F9FA`, I will use it instead.